lib/util.js

```diff
@@ -242,6 +243,10 @@ inspect.styles = Object.assign(Object.create(null), {
   // "name": intentionally not styling
   'regexp': 'red'
 });
+// Magenta has problems with the deafult configuration of Powershell
+// Ref: https://github.com/nodejs/node/issues/14243
+if (process.platform === 'win32')
+  inspect.styles.date = 'magenta;bold';
```

Comment:
I'm saying a proper "fix" would be to intercept any write to stdout and replace the non-bold sequence with the bold one.